dwl.c

```diff
@@ -84,6 +85,7 @@ typedef struct {
 	int bw;
 	unsigned int tags;
 	int isfloating;
+	int dirty;
```

Comment:
Maybe `resizing` instead - "dirty" could mean a lot more than just that.

Reply:
Changed to resize, which is the configure serial of the outstanding operation.